src/scss/components/_cards.scss

```diff
@@ -92,9 +93,22 @@ $card-section-collapsible-paddingRight--aboveMid: 60px;
                 }
             }
 
+            .c-card-section--collapsible--noPad {
+                &.c-card-section {
```

Comment:
How come this needs to be included?  Is it to increase the specificity?

Reply:
It was left in after a refactor — will remove.